### PR TITLE
Feat/greenlight shellcheck

### PR DESCRIPTION
**Pull request by linuxbandit**
for a mistake it is built on top of #131, so just merge that first

### CODE REVIEW

**Review by serge1peshcoff — APPROVED** [review]

File: helper.sh [hunk 1]
```diff
@@ -220,11 +194,12 @@ if [[ "$#" -ge 1 ]]; then
             --docker) docker=true; ((command_num++)); shift ;;
             -v) verbose=true; shift ;;
 
-            --) shift ; arguments+=$@; break ;;
+            --) shift ; arguments+=("${@}"); break ;;
```
Comment (serge1peshcoff) [inline helper.sh:197]: one of the reasons to hate bash

Comment (serge1peshcoff) [reply helper.sh:197]: we've completely ditched it and replaced it with Python, it may be a good idea to do it as well

Comment (serge1peshcoff) [reply helper.sh:197]: (somewhere in the future, this is just a food for thought)

Comment (linuxbandit) [reply helper.sh:197]: agree, i was thinking about grunt (js). It is basically a replacement for make and for the helper script at once

Comment (serge1peshcoff) [reply helper.sh:197]: I'd rather go for gulp then, everybody uses it now

Comment (linuxbandit) [reply helper.sh:197]: > Each plugin for Gulp is written with the goal of doing one thing very well.

Unix philosophy? I'm in

Comment (linuxbandit) [reply helper.sh:197]: @WikiRik can you write it off as a low priority task?

Comment (serge1peshcoff) [reply helper.sh:197]: I've also used gulp before webpack (they are slightly different though: gulp is a task manager when webpack is a frontend build tool), worked flawlessly for me

Comment (WikiRik) [reply helper.sh:197]: > @WikiRik can you write it off as a low priority task?

What exactly?

**Pull request merged**